### PR TITLE
Update to new Rekcurd `v0.5.0a`

### PR DESCRIPTION
**Pull request by keigohtr**
## What is this PR for?

Apply https://github.com/rekcurd/rekcurd-python/pull/36

## This PR includes

- Update `python/sklearn-digits/`

## What type of PR is it?

Feature

## What is the issue?

N/A

## How should this be tested?

N/A

### CODE REVIEW

File: python/sklearn-digits/app.py [hunk 1]
```diff
@@ -19,51 +18,42 @@
...
+            result.label: Label. One of string/bytes/arr[int]/arr[float]/arr[string]
+            result.score: Score. One of float/arr[float]
+            result.option: Miscellaneous. dict
         """
         try:
-            if option is None:
-                option = {}
-            label_predict = self.predictor.predict(
+            label_predict = predictor.predict(
                 np.array([input], dtype='float64')).tolist()
```
Comment (yuki-mt) [inline python/sklearn-digits/app.py]: needs to change `input` to `idata`

File: python/sklearn-digits/app.py [hunk 2]
```diff
@@ -107,22 +98,29 @@ def evaluate(self, file_path: str) -> Tuple[EvaluateResult, List[EvaluateResultD
...
+        :return rtn: Return results. Generator[EvaluateDetail, None, None]
+            rtn.input: Input data. PredictInput, one of string/bytes/arr[int]/arr[float]/arr[string]
+            rtn.label: Predict label. PredictLabel, one of string/bytes/arr[int]/arr[float]/arr[string]
+            rtn.result: Predict detail. EvaluateResultDetail
         """
         try:
-            for i, (correct_label, data) in enumerate(self.__generate_eval_data(file_path)):
-                yield EvaluateDetail(input=data, label=correct_label, result=results[i])
+            for i, (correct_label, data) in enumerate(self.__generate_eval_data(filepath)):
+                yield EvaluateDetail(input=data, label=[correct_label], result=details[i])
```
Comment (yuki-mt) [inline python/sklearn-digits/app.py]: we should match label's types. (label's type in here is array of string and label's type in `predict` method is string)

File: python/sklearn-digits/app.py (hunk 1 above)
Comment (keigohtr) [reply python/sklearn-digits/app.py]: I have updated!

File: python/sklearn-digits/app.py (hunk 2 above)
Comment (keigohtr) [reply python/sklearn-digits/app.py]: I have updated!

**Comment by keigohtr** [comment]
@yuki-mt Thank you for your review!
I have updated, could you check it again please?

File: .gitignore [hunk 3]
```diff
@@ -103,8 +103,9 @@ ENV/
 # PyCharm
 .idea/
 
-# ML Model
-model/
+# Rekcurd
+rekcurd-model/
```
Comment (yuki-mt) [inline .gitignore:108]: this example run as local mode, so LOCAL_MODEL_DIR is `model/`, not `rekcurd-model` .
(so you should ignore  `model/` not `rekcurd-model/`

Comment (keigohtr) [reply .gitignore:108]: I have added `model/`. `rekcurd-model/` is also used when we don't use `local` mode.

**Review by yuki-mt — APPROVED** [review]
LGTM!

**Pull request merged**